src/main/java/hudson/plugins/jira/JiraRestService.java

```diff
@@ -170,7 +165,8 @@ public Issue getIssue(String issueKey) {
 
     public List<IssueType> getIssueTypes() {
         try {
-            return Lists.newArrayList(jiraRestClient.getMetadataClient().getIssueTypes().get(timeout, TimeUnit.SECONDS));
+            return StreamSupport.stream(jiraRestClient.getMetadataClient().getIssueTypes().get( timeout, TimeUnit.SECONDS ).spliterator(), false ).
```

Comment:
what's the purpose of using streams here?

Reply:
just to get rid of Guava with a one liner
I can use this as well 
```
            List<IssueType> result = new ArrayList<>( );
            jiraRestClient.getMetadataClient().getIssueTypes().get( timeout, TimeUnit.SECONDS ).forEach(result::add);
            return result;
```
Not sure which one is the best solution